src/client.rs

```diff
@@ -35,6 +35,9 @@ impl Client {
     pub fn builder(url: impl AsRef<str>) -> Result<Builder<MissingCred>> {
         Builder::new(url)
     }
+    pub fn build_region(url: impl AsRef<str>) -> Result<Builder<MissingCred>> {
+        Builder::new(url)
+    }
```

Comment:
I guess we were supposed to take an AWS region in parameter here + it’s missing the doctest